src/mcts/search.cc

```diff
@@ -492,17 +513,19 @@ EdgeAndNode Search::GetBestChildNoTemperature(Node* parent) const {
 // Returns a child chosen according to weighted-by-temperature visit count.
 EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
                                                 float temperature) const {
+  // Only works with root node because of draw score handling.
+  assert(parent == root_node_);
```

Comment:
Rather than assert, why not remove the input parameter and rename to GetBestChildOfRootWithTemperature?

Reply:
Done.